pkg/api/v1/types.go

```diff
@@ -589,6 +594,15 @@ type CephFSVolumeSource struct {
 	ReadOnly bool `json:"readOnly,omitempty"`
 }
 
+// FlockerVolumeSource represents a Flocker volume attached to a kubelet's host machine and exposed to the pod for its usage.
+type FlockerVolumeSource struct {
+	// Required: the volume name. This is going to be store on metadata -> name on the payload for Flocker
+	DatasetName string `json:"datasetName"`
+
+	// Optional: the size of the volume in human readable format.
+	Size string `json:"size"`
```

Comment:
Please use resource.Quantity as this type. (Find other usages elsewhere in this file.)

Meta: it seems odd to specify a size here. Other volumes (including persistent volumes) don't specify a size. Why is this volume different?